account_payment_term_cutoff_day/models/account_payment_term_line.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2025 Quartile
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import fields, models
+
+
+class AccountPaymentTermLine(models.Model):
+    _inherit = "account.payment.term.line"
+
+    cutoff_day = fields.Integer(
```

Comment:
A constraint to prevent the input of a random number could be helpful, but I guess it's also fine without it.